test/e2e-go/features/transactions/asset_test.go

```diff
@@ -80,45 +77,45 @@ func TestAssetValidRounds(t *testing.T) {
 	firstValid = 0
 	lastValid = 0
 	validRounds = cparams.MaxTxnLife + 1
-	firstValid, lastValid, err = client.ComputeValidityRounds(firstValid, lastValid, validRounds)
+	firstValid, lastValid, lastRound, err = client.ComputeValidityRounds(firstValid, lastValid, validRounds)
 	a.NoError(err)
-	a.Equal(params.LastRound+1, firstValid)
+	a.Equal(lastRound+1, firstValid)
```

Comment:
what exactly changed here? Doesn't lastRound always get the value of params.LastRound?

Reply:
params might be requested at round X but ComputeValidityRounds called at round X+1, so it returns the round it used as a calculation base